src/Log/Engine/FileLog.php

```diff
@@ -116,7 +116,6 @@ public function __construct(array $config = [])
      */
     public function log($level, $message, array $context = []): void
```

Comment:
```suggestion
    public function log($level, string $message, array $context = []): void
```

Can we do this?

Reply:
No we can't because `LoggerInterface::log()` doesn't have the `string` typehint.

Reply:
Ah yes, the PSR versioning issue.